### DISCUSSION TITLE
Feature request: selecting themes/tabs without automatically selecting the scenery in it

### DISCUSSION THREAD

**Discussion by Liampie** [OpenRCT2/OpenRCT2#15758] [Feature requests]
Currently when you select a scenery tab, the game tries to add all the scenery that belongs to that tab to your savegame. Many people use tabs simply as a means to organise objects, not to select them. This is one thing where Parkdat still trumps OpenRCT.

A checkbox in the object selection window that says "Select scenery with theme" or whatever that can be switched off to just select the tab itself would be very helpful.

**Comment by RWErct2** [comment]
I like that idea!

**Comment by Gymnasiast** [comment]
This is a power user feature, IMO. I think holding Ctrl to only select the group itself is sufficient.

**Comment by Broxzier** [comment]
These 'power features' need to be written down somewhere. Maybe on the wiki.

**Comment by Liampie** [comment]
Instead of hiding functions like that, maybe put a 'power user' checkbox in the options that reveals more checkboxes and functions all around the game, like this one. I get that you don't want to clutter the object selection window with a checkbox for this, but the holding ctrl thing is so hidden that plenty of people will probably not find out at all unless they accidentally stumble upon it while reading github?
